bindings/go/plugin/manager/manager.go

```diff
@@ -64,7 +65,7 @@ func WithIdleTimeout(d time.Duration) RegistrationOptionFn {
 // RegisterPlugins walks through files in a folder and registers them
 // as plugins if connection points can be established. This function doesn't support
 // concurrent access.
-func (pm *PluginManager) RegisterPlugins(ctx context.Context, dir string, opts ...RegistrationOptionFn) error {
+func (pm *PluginManager) RegisterPlugins(ctx context.Context, config *v1.Config, dir string, opts ...RegistrationOptionFn) error {
```

Comment:
@jakobmoellerdev I imagined this to be called from CLI side or wherever and then a parsed config would be given to the registration. From there, using Filter, we get the right config to the right plugin. This sounds simple enough....

Reply:
I was hoping the config would be passed as a RegistrationOptionFn